templates_eventbridge/EventBridge.yaml

```diff
@@ -76,4 +76,4 @@ Resources:
           Arn: !Ref EventBusARN
           RoleArn: !GetAtt
             - EventBridgeRole
-            - Arn
+            - Arn        
```

Comment:
`make lint` should fix this extra spaces.
its odd the ci did not detect this

Reply:
removed the extra space.